src/region/RegionAnnotations.tsx

```diff
@@ -34,7 +39,17 @@ export default class RegionAnnotations extends React.Component<Props, State> {
         scale: 1,
     };
 
-    state: State = {};
+    constructor(props: Props) {
+        super(props);
+
+        let editorState = EditorState.createEmpty(mentionDecorator);
+        if (props.staged && props.staged.message) {
+            const contentState = ContentState.createFromText(props.staged.message);
+            editorState = EditorState.createWithContent(contentState, mentionDecorator);
```

Comment:
I don't think the top-level component should be involved in creating the form state. Can we isolate these details in the form?